community/cypher/cypher-compiler-3.0/src/test/scala/org/neo4j/cypher/internal/compiler/v3_0/planner/logical/idp/IDPQueryGraphSolverTest.scala

```diff
@@ -490,7 +490,7 @@ class IDPQueryGraphSolverTest extends CypherFunSuite with LogicalPlanningTestSup
       new IDPSolverConfig {
         override def solvers(queryGraph: QueryGraph): Seq[(QueryGraph) => IDPSolverStep[PatternRelationship, LogicalPlan, LogicalPlanningContext]] =
           ExpandOnlyWhenPatternIsLong.solvers(queryGraph)
-        override def iterationDurationLimit: Long = 100
+        override def iterationDurationLimit: Long = 50
```

Comment:
I see no problem with that. You're just tightening the time constraint. Just curious why you needed to do this? Was the test being flakey in some way? Seems unrelated to the PR.